internal/engine/experiment/nwebconnectivity/control.go

```diff
@@ -0,0 +1,124 @@
+package nwebconnectivity
```

Comment:
*Code organization suggestion* since the nwebconnectivity will likely need to use the same primitives of its control implementation, and the control implementation itself is private, how about having most of the code inside of nwebconnectivity and exporting only the bare minimum required by the new control code?

Reply:
what data structures are you referring to?

Reply:
It seems to me that most of the building blocks are the same and the difference is quite little, but maybe that's not the case. I propose to do nothing for now. If we need to merge/move things, we'll notice.